tests/unit/TickingMockClockTest.php

```diff
@@ -32,9 +32,9 @@ public function it_returns_time_that_has_ticked(): void
         // Immutable variant.
 
         $sample1 = $clock->now();
-        \usleep(0);
+        \usleep(1);
```

Comment:
Unrelated, but makes the tests less flaky. Regularly fails with `0` for me 😠

Reply:
![6u77tkmyaomz.jpg](https://github.com/user-attachments/assets/71bee210-d7f9-4bad-a023-8f959c55227f)